namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-visits-booking-production/resources/elasticache.tf

```diff
@@ -0,0 +1,23 @@
+module "ec-cluster-prison-visits-booking-staff" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=3.0"
```

Comment:
Could you please use the latest version of that module ? 3.1
https://github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster/releases/tag/3.1